app/assets/javascripts/vote.js

```diff
@@ -0,0 +1,36 @@
+
+$(document).ready(function () {
+  $(".like").submit(function (event) {
+    event.preventDefault();
```

Comment:
Cool stuff! Consider breaking this into smaller functions to make it more readable (or, day I say it... react?), but this looks great.